File: Java/Math/ExcelColumnNumber.java
public class Solution {
	public int titleToNumber(String a) {
	    int len = a.length()-1;
	    int res = 0;
	    for(int i = 0; i <= len; i++){
	        char c = a.charAt(i);
	        res = 26 * res + (c - 'A' + 1);
	    }
	    return res;
	}
}

File: JS/Math/ExcelColumnNumber.js
module.exports = {
	//param A : string
	//return an integer
	titleToNumber : function(A){
        var res = 0;
        for(var i = 0; i < A.length; i++){
            res = res * 26 + (A.charCodeAt(i) - 64);
        }
        return res;
	}
};
